Makefile

```diff
@@ -13,7 +13,7 @@
 
 include Makefile.common
 
-.PHONY: all clean test gotest server dev benchkv benchraw check checklist parser tidy ddltest
+.PHONY: all clean test gotest server dev benchkv benchraw check checklist parser tidy ddltest br_web
```

Comment:
Is there any reason only `br_web` is added?

Reply:
Fixed, `br_web` can be removed here, We use build_tools to generate the binary.

Reply:
Sure~